ci/travis/check_symbol_collisions.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
Any reason why this can't be added as a normal `pytest` test?

Reply:
It's. a normal pytest.

Reply:
Ah nice. Can we put it in `python/ray/tests` or does that not work because it'll force ray to be imported first?

Reply:
Moved there and added to the BUILD file.